platform/mbed_lib.json

```diff
@@ -159,6 +159,10 @@
         "ARM_MUSCA_A1_S": {
             "stdio-baud-rate": 115200
         },
+        "CY8CPROTO_064_SB": {
+            "stdio-baud-rate": 115200,
+            "default-serial-baud-rate": 115200
```

Comment:
Why do we need to set `default-serial-baud-rate`? Aren't we only using DAPLINK serial which should be mapped to `stdio`? If yes, then setting `stdio-baud-rate` should suffice right?

Reply:
I was advised to keep it same by @ashok-rao